app/views/orders/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Past Orders</h1>
+<ul>
+  <% @order.each do |order| %>
```

Comment:
@jtruong2 @order should probably be @orders. It is consistent through the controller now so it works, it just seems like something we could clean up.

Nicely done on Line: 6. A potential refactor could exist on the Order model as `delivery_address` or something along those lines just so we could call `order.delivery_address` once in the view and have it show all of that info. Maybe something like this...

```
class Order < ApplicationRecord
    belongs_to :user

    def delivery_address
        "#{self.order_address} #{self.order_city}, #{self.order_state} #{self.order_zipcode}"
    end
end
```